lib/Fmt_ast.ml

```diff
@@ -541,6 +541,14 @@ let sequence_blank_line c (l1 : Location.t) (l2 : Location.t) =
       loop l1.loc_end (Cmts.remaining_before c.cmts l2)
   | `Compact -> false
 
+let fmt_quoted_string c key ext s strloc delim =
+  let delim = Option.value delim ~default:"" in
+  Cmts.fmt c strloc
```

Comment:
Are there some comments that get attached here?

Reply:
It looks like there is none, I simplified the code.